app/models/comment.rb

```diff
@@ -1,5 +1,4 @@
 class Comment < ApplicationRecord
   belongs_to :post
-  belongs_to :user
```

Comment:
I think comments should belong to user. At the moment it seems that comments are anonymous. We should aim to have the user who commented visible with the comment.